rcp/org.testeditor.rcp4.tcltestrun.tests/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncherTest.xtend

```diff
@@ -0,0 +1,48 @@
+package org.testeditor.rcp4.tcltestrun
+
+import java.io.File
+import java.nio.file.Files
+import org.eclipse.core.resources.IProject
+import org.eclipse.core.runtime.IPath
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.runners.MockitoJUnitRunner
+
+import static org.junit.Assert.*
+import static org.mockito.Mockito.*
+
+@RunWith(MockitoJUnitRunner)
```

Comment:
We have a class `AbstractTest` in the common.testing bundle that takes care of this. We should inherit from this one.